tools/releases/dockerfiles/Dockerfile.kuma-dp

```diff
@@ -1,28 +1,20 @@
-ARG BASE_IMAGE_ARCH=amd64
-
 FROM debian:11 as envoy
 ARG ENVOY_VERSION
 ARG ARCH
 
-ADD /build/artifacts-linux-$ARCH/envoy/envoy-$ENVOY_VERSION-alpine /envoy
+COPY /build/artifacts-linux-$ARCH/envoy/envoy-$ENVOY_VERSION-alpine /envoy
```

Comment:
@jakubdyszkiewicz may add something here but I think this might not have the same name in the future. Maybe extracting the full envoy name as an ARG instead of ENVOY_VERSION would be better

Reply:
I'm ok leaving it as is.